docs/serving/installing-istio.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Performing a Custom Istio Installation"
+weight: 15
+type: "docs"
+---
+
+Use this guide to perform a custom installation of Istio for use with Knative.
+
+## Download Istio
+
+Run below command to download Istio.
+```shell
+# Download and unpack Istio
+ISTIO_VERSION=1.1.3
+DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz
+
+wget $DOWNLOAD_URL
+tar xzf istio-${ISTIO_VERSION}-linux.tar.gz
+
+cd istio-${ISTIO_VERSION}
+```
+
+## Install Istio CRDs 
```

Comment:
Just to make sure I understand, after this step for installing the CRDs, you then have to go and follow the instructions for either "Istio with Sidecar Injector" or "Istio with no Sidecar Injector," right? You're definitely not done at this point?